### PR TITLE
Add SQS queue for star alignment jobs

### PR DESCRIPTION
**Pull request by reisingerf**
This is the only place I need to add the SQS, right?

### CODE REVIEW

File: terraform/stacks/umccr_data_portal/pipeline/oncoanalyser.tf [hunk 1]
```diff
@@ -50,6 +50,23 @@ resource "aws_cloudwatch_event_target" "oncoanalyser" {
...
+}
+
+resource "aws_ssm_parameter" "sqs_star_alignment_queue_arn" {
+  name  = "${local.ssm_param_key_backend_prefix}/sqs_star_alignment_queue_arn"
+  type  = "String"
+  value = aws_sqs_queue.star_alignment_queue.arn
+  tags  = merge(local.default_tags)
+}
+
+# --- oncoanalyser wts queue
```
Comment (victorskl) [inline terraform/stacks/umccr_data_portal/pipeline/oncoanalyser.tf:68]: Could you align this comment accordingly, pls. May be update the one at the top to star...

**Comment by victorskl** [comment]
[Portal Release - Ocicat: Implement Portal automation integration for oncoanalyser](https://trello.com/c/sZFv7FhZ/1488-portal-release-ocicat-implement-portal-automation-integration-for-oncoanalyser)

**Review by victorskl — APPROVED** [review]
Yup. LGTM! Minor comment.

File: terraform/stacks/umccr_data_portal/pipeline/oncoanalyser.tf (hunk 1 above)
Comment (reisingerf) [reply terraform/stacks/umccr_data_portal/pipeline/oncoanalyser.tf:68]: Ups!

**Comment by victorskl** [comment]
Good to merge (GTM), Flo.!

**Pull request merged**